kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/AccordionContainer.vue

```diff
@@ -24,21 +57,34 @@
 
 <script>
 
+  import { enhancedQuizManagementStrings } from 'kolibri-common/strings/enhancedQuizManagementStrings';
+
   export default {
     name: 'AccordionContainer',
-    data() {
+    setup() {
+      const { expandAll$, collapseAll$ } = enhancedQuizManagementStrings;
       return {
-        expandedItemIds: [],
+        expandAll$,
+        collapseAll$,
       };
     },
-    watch: {
-      expandedItemIds() {
-        this.$emit('toggled', this.expandedItemIds);
```

Comment:
By handling the collapse all and expand all within the AccordionComponent, I think we can handle the functionality of the expandedItemIds within the component without having to expose the entire array, and just expose the `isItemExpanded` method.

Reply:
Agreed 👍🏻